app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/components/ide.tsx

```diff
@@ -0,0 +1,74 @@
+/** @file  */
+import * as react from "react";
+
+import * as service from "../service";
+
+const IDE_CDN_URL = "https://ensocdn.s3.us-west-1.amazonaws.com/ide";
+
+interface Props {
+  project: service.Project;
+  backendService: service.Backend;
+}
+
+declare global {
+  const enso: { main: (config: object) => void; };
+}
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
Why do we need to disable the lint?

Reply:
currently the `naming-convention` lint is explicitly disabled with a comment without further explanation for every react component, since react components use PascalCase which isn't allowed by `naming-convention` by default

Reply:
Let's fix the linter config then :)

Reply:
I feel like that would belong in a separate PR? since it would involve changing everywhere else that has the naming-convention lint disabled

Reply:
In a separate PR we can fix the existing places, but disabling lints in a new code is always a no-go for us if this is not really necessary / this is not the correct solution. Unfortunately, there exist a code with bad things. It was probably not caught during previous PR reviews. Anyway, I believe that the new code should not have it if we can fix it simply. I understand that by fixing the linter config you do not need to modify the old code, right?

Reply:
right